server/bin/unittests

```diff
@@ -314,6 +316,12 @@ function _setup_state {
     _state_config="${_tdir}/state/${1}.config/pbench-server.cfg"
     if [ ! -e ${_state_config} ]; then
         _state_config="${_tdir}/state/config/pbench-server.cfg"
+    else
+        # the testcase-specific config file includes the global state config file
+        # but under the name "state-pbench-server.cfg" to prevent conflicts. It also
+        # gets the defaul section below, so we have to include the install-dir line in it
+        # and delete it from the global state config file.
+       sed 1d "${_tdir}/state/config/pbench-server.cfg" > ${_testtmp}/state-pbench-server.cfg
```

Comment:
Seems like the `sed` command is indented 3 spaces instead of 4 ...

Also, why not just copy this state file to the same location as the default config file?

```
        sed 1d "${_tdir}/state/config/pbench-server.cfg" > ${_testopt}/lib/config/state-pbench-server.cfg
```

And then clean it up explicitly below when the test is done?
```